tutorials/shaders/shader_reference/canvas_item_shader.rst

```diff
@@ -217,12 +220,12 @@ Light processor functions work differently in Godot 4.x than they did in Godot
 3.x. In Godot 4.x all lighting is done during the regular draw pass. In other
 words, Godot no longer draws the object again for each light.
 
-Use render_mode ``unshaded`` if you do not want the light processor function to
-run. Use render_mode ``light_only`` if you only want to see the impact of
+Use render mode ``unshaded`` if you do not want the ``light()`` function to
```

Comment:
```suggestion
Use the ``unshaded`` render mode if you do not want the ``light()`` function to
```
Or keep the underscore IMO, this reads ungrammatical to me

Reply:
I think either "the render mode ``unshaded``", or "the ``unshaded render mode" are good. But "render mode ``unshaded``" without a "the" is bad. I'd like to avoid "render_mode" without code styling in body text.

Reply:
"the render mode `unshaded`" is ungrammatical I'd say, so pick the swapped order